install.py

```diff
@@ -1962,10 +2131,30 @@ def convert_to_int(string):
                 value = string
             return value
 
-        version_info_list = [convert_to_int(s) for s in version_info_list]
+        version_info_list = (convert_to_int(s) for s in version_info_list)
 
         return tuple(version_info_list)
 
+    @staticmethod
+    def _version_cmp(operation):
```

Comment:
`def _version_cmp` has `_method`. You wanted to implement a kind of private method, right? Or you did put the heading the `_` by typo?
If it means a private method, could you move it to after a public method `version_equal`? This is like `class Python` in `install.py`.

Reply:
Yes, the `_` prefix is intentional. I'll move it after the public methods.

Reply:
Sure, thanks for moving it.